.gitignore

```diff
@@ -0,0 +1,3 @@
+# what is the function of the .deployed file?
```

Comment:
It indicates that a component (anaconda, eups, etc.) has been installed already if `bin/deploy` is re-executed.